core/src/main/java/io/substrait/extendedexpression/ExtendedExpressionProtoConverter.java

```diff
@@ -0,0 +1,66 @@
+package io.substrait.extendedexpression;
+
+import io.substrait.expression.Expression;
+import io.substrait.expression.proto.ExpressionProtoConverter;
+import io.substrait.extension.ExtensionCollector;
+import io.substrait.proto.AggregateFunction;
+import io.substrait.proto.ExpressionReference;
+import io.substrait.proto.ExtendedExpression;
+import io.substrait.type.proto.TypeProtoConverter;
+
+/** Converts from {@link ExtendedExpression} to {@link ExtendedExpression} */
```

Comment:
minor: As is these docs links are the same class. If we fully qualify them it's clearer that they are distinct ExtendedExpression classes.
java
```
/** Converts from {@link io.substrait.extendedexpression.ExtendedExpression} to {@link io.substrait.proto.ExtendedExpression} */
```